Project.toml

```diff
@@ -20,6 +21,7 @@ DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
 EnumX = "4e289a0a-7415-4d19-859d-a7e5c4648b56"
 FindFirstFunctions = "64ca27bc-2ba2-4a57-88aa-44e436879224"
 FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
This is useful for interaction, but I can imagine CairoMakie is enough for quarto static plots?